### ISSUE TITLE
USENET integration

### ISSUE THREAD

**Issue by Littlemanjr2** [tommyvange/ArrStalledHandler#9]
Can you make this work with USENET downloads as well? I'd like this to remove and blocklist downloads that cant be imported automatically that just sit in the queue.

**Comment by tommyvange** [comment]
> Can you make this work with USENET downloads as well? I'd like this to remove and blocklist downloads that cant be imported automatically that just sit in the queue.

What does this look like? I have not personally experienced an issue like. Do you mean downloads that require manual import?

Could you provide some screenshots from SabNZB or NZBget as well as screenshots from Radarr or Sonarr.

**Comment by tommyvange** [comment]
> When a DL completes but for a few reasons it can't import automatically.
> 
> ![Screenshot 2024-11-25 065430](https://private-user-images.githubusercontent.com/26419886/389581530-f1d6f938-5060-499c-9710-ac4e8bae5926.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NDg2NjQsIm5iZiI6MTczMjU0ODM2NCwicGF0aCI6Ii8yNjQxOTg4Ni8zODk1ODE1MzAtZjFkNmY5MzgtNTA2MC00OTljLTk3MTAtYWM0ZThiYWU1OTI2LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI1VDE1MjYwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA5MzBhZDc1NmUwZTk3YmY0Y2FiOTBkMTM1ZDQxMDIyNDQ2NmUxMGE0YTc3YmY4ZTdkNzc0YzU1NGMwYTc5ZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.cIGks7-j5XuEn-vZzsG3JbEifuY3OaltRwFDX86K_ag) ![Screenshot 2024-11-25 065352](https://private-user-images.githubusercontent.com/26419886/389581543-71ade781-87c2-4240-ae5a-befb1ce378eb.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzI1NDg2NjQsIm5iZiI6MTczMjU0ODM2NCwicGF0aCI6Ii8yNjQxOTg4Ni8zODk1ODE1NDMtNzFhZGU3ODEtODdjMi00MjQwLWFlNWEtYmVmYjFjZTM3OGViLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDExMjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMTI1VDE1MjYwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWNhODFjNzE4ZGZjNjBjMzA2NDNkNGMyNWFjYTFkMGM5OGQxOTgyZTQyZGU1YTcxZDU5Yzg0YzAxNzY1NjZkNjEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.QVYJoZCpNdGkigBXY744F5eC3asdBeEoIc3XJJts0OE)

 Alright, i understand. I will put it on my to-do list.

**Comment by Littlemanjr2** [comment]
Hell yeah, thanks man.

**Comment by WeterPeter** [comment]
yes please!

**Comment by FelixHutnyk** [comment]
This would be amazing!

**Comment by Willizxy** [comment]
This would be very useful

**Comment by Willizxy** [comment]
This would be very useful

**Comment by adalius** [comment]
Agreed! My biggest failure message is 'No files found eligible for import' because the file inside the nzb will have a bad extension (like .rar) or some other issue and it drives me nuts. Sometimes I have to blocklist and search 15-20x before it finds a valid version.
